kotlin/README.md

```diff
@@ -0,0 +1,29 @@
+# Algolia's Kotlin demo app
+
+## Kotlin setup
+
+1. install jdk `brew install openjdk`
+2. install gradle `brew install gradle`
+3. prepare gradle `gradle wrapper`
+4. install dependencies `./gradlew dependencies`
+
+### VSCode
+
+1. Install maven `brew install mvn`
+2. add the ["Kotlin" extension](https://marketplace.visualstudio.com/items?itemName=fwcd.kotlin)
+
+## How to use the project
+
+This project aims to show how to use the Kotlin Algolia API client.
+
+First, build the project
+
+```shell
+./gradlew build
+```
+
+Then, run the project
+
+```shell
+??????????
```

Comment:
not sure, I didn't get that far

Reply:
I've updated the guide, please take a look